components/tools/OmeroWeb/omeroweb/webgateway/marshal.py

```diff
@@ -134,9 +134,18 @@ def imageMarshal(image, key=None):
 
     # big images
     tiles = image._re.requiresPixelsPyramid()
-    width, height = image._re.getTileSize()
-    levels = image._re.getResolutionLevels()
-    zoomLevelScaling = image.getZoomLevelScaling()
+    rv['tiles'] = tiles
+    if (tiles):
```

Comment:
While I agree that more testing is generally better, I don't think that moving a couple of lines out of a AttributeError try catch requires a new test when those lines can't throw AttributeError. Also, it's not really a unit test because this code requires a ImageWrapper (Big Image) with rendering engine etc.